Sources/Tuvi.Core.Entities/Account.cs

```diff
@@ -53,8 +53,28 @@ public Account()
         public int Id { get; set; }
 
         [SQLite.Ignore]
-        public EmailAddress Email { get; set; }
-        [SQLite.Indexed]
+        public EmailAddress Email
+        {
+            get
+            {
+                if (_Email is null && EmailAddress != null)
+                {
+                    _Email = new EmailAddress(EmailAddress, EmailName);
+                }
+                return _Email;
+            }
+            set
+            {
+                EmailAddress = value?.Address;
+                EmailName = value?.Name;
+                _Email = value;
+            }
+        }
+        private EmailAddress _Email;
+        public string EmailAddress { get; set; }
+        public string EmailName { get; set; }
```

Comment:
`_Email.Name` and `EmailName` can contain different values
`public string EmailName=> _Email?.Name`

Reply:
EmailName is a field to be stored in the database

Reply:
reset value `_Email = null` in setter

Reply:
Done